pkg/applicationserver/applicationserver.go

```diff
@@ -126,6 +127,18 @@ func New(c *component.Component, conf *Config) (as *ApplicationServer, err error
 		formatters: payloadFormatters(map[ttnpb.PayloadFormatter]messageprocessors.PayloadEncodeDecoder{
 			ttnpb.PayloadFormatter_FORMATTER_JAVASCRIPT: javascript.New(),
 			ttnpb.PayloadFormatter_FORMATTER_CAYENNELPP: cayennelpp.New(),
+			ttnpb.PayloadFormatter_FORMATTER_REPOSITORY: devicerepository.New(
+				map[ttnpb.PayloadFormatter]messageprocessors.PayloadEncodeDecoder{
+					ttnpb.PayloadFormatter_FORMATTER_JAVASCRIPT: javascript.New(),
```

Comment:
You may share the processor with the one above. It won't have any effect now, but in case we add some sort of caching later, it will help.